css/main.css

```diff
@@ -56,19 +61,177 @@ textarea {
    Author's custom styles
    ========================================================================== */
 
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.black {
+  color: #000;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.main {
+  display: flex;
+  overflow: hidden;
+  font: normal 16px sans-serif;
+  width: 100%;
+  height: 100%;
+}
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  background: #0e171d;
+  color: #fff;
+  width: 300px;
+  height: 100%;
+  float: left;
+}
+
+.logo-motorpool {
+  background-image: url("../img/logo.png");
```

Comment:
Ideally, you'd want to make this logo file twice as big and use `background-size` to shrink it to half size — this way, it won't be pixelated on retina devices.

(There are a number of tactics for dealing with this problem, though, including just inlining the image)